packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -168,6 +173,10 @@ const AppModals = ({
         ComponentToLoad = <NeedRealAccountForCashierModal />;
     }
 
+    if (is_additional_kyc_info_modal_open) {
+        ComponentToLoad = <AdditionalKycInfoModal />;
+    }
```

Comment:
@shahzaib-deriv Would it be possible to render the modal where needed and let the parent control it instead of adding more states to stores? 🤔

Reply:
@farzin-deriv this modal needs to be at the very core as it should be accessible from the notification action. Is there a better place to handle such scenario?